packages/devtools/cli/README.md

```diff
@@ -29,6 +29,14 @@ Agent is a name for DXOS Client which is run by CLI in the background in the dae
 1. Run `launchctl unload -w ~/Library/LaunchAgents/org.dxos.agent.plist`
 2. Remove `~/Library/LaunchAgents/org.dxos.agent.plist`
 
+## Install agent as a `systemd` service (Linux users)
+1. Install DXOS CLI with steps in [Installation](#Instalation) section.
+1. Copy `./init-templates/dxos-agent.service` and `./init-templates/pre-dxos-agent.service` -> `~/.config/systemd/user/`
```

Comment:
Numbering

Reply:
Markdown will render the numbers automatically, so I personally default to just setting them to `1.` so I don't have to change them manually if I add or remove things.

But if the repo style is to set them there, I can adapt...